### ISSUE TITLE
Share pdf via email when link doesn't end with ".pdf"

### ISSUE THREAD

**Issue by Temeteron** [EddyVerbruggen/SocialSharing-PhoneGap-Plugin#904]
Hi,

I am trying to share a pdf file from a google drive link which doesn't contain the "*.pdf". 

It attaches a file that isn't recognized and subsequently cannot be opened.

I tried to share it with the different available methods with no result. Probably it is not supported.

One approach i tried is the following.
```
var options = {
      message: "some text",
      subject: "my subject",
      files: ["https://drive.google.com/file/d/0B7CKnJulpC4_UDE0TXN3SmhqTVFCVDRESXluRHpHWkVINVZF"],
}

```

Thanks in advance!

**Comment by EddyVerbruggen** [comment]
Is this problem observed on ios, android, or both?

And is it specific to this pdf, or any?

**Comment by Temeteron** [comment]
observed on both, about any pdf from a google drive link

**Comment by agoldenberg** [comment]
Yeah it looks like file extension is a requirement. I'm trying the same thing with a QR code from https://api.qrserver.com/v1/create-qr-code/?size=150x150&data=abc123 and it won't fetch the image either.
